src/mbgl/renderer/painter.cpp

```diff
@@ -89,7 +89,7 @@ Painter::Painter(gl::Context& context_, const TransformState& state_, float pixe
     tileBorderSegments.emplace_back(0, 0, 4, 5);
     rasterSegments.emplace_back(0, 0, 4, 6);
 
-    gl::debugging::enable();
+    context.enableDebugging();
```

Comment:
We always need it, but currently, there's no guarantee that the OpenGL context will be active when the `Context` object is created (the same goes for extension initialization). We could refactor this code to wait with initialization of the `Backend` object until we know that the OpenGL context is active, e.g. by lazily generating it in the `Backend::getContext()` call.